simulariumio/physicell/physicell_converter.py

```diff
@@ -316,20 +312,30 @@ def _get_trajectory_data(
                     ] = (subcell_positions[subcell_index] - center)
                     result.subpoints[time_index][agent_index][
                         sp_index + VALUES_PER_3D_POINT
-                    ] = (
-                        input_data.meta_data.scale_factor
-                        * PhysicellConverter._radius_for_volume(
-                            discrete_cells[time_index]["total_volume"][cell_index]
-                        )
+                    ] = PhysicellConverter._radius_for_volume(
+                        discrete_cells[time_index]["total_volume"][cell_index]
                     )
                 agent_index += 1
             result.n_agents[time_index] = agent_index
+
+        if input_data.meta_data.scale_factor is None:
+            # If scale factor wasn't provided, calculate one
+            scale_factor = TrajectoryConverter.calculate_scale_factor(result)
+        else:
+            scale_factor = input_data.meta_data.scale_factor
+        for index in range(dimensions.total_steps):
+            # do not scale radii for subcells
```

Comment:
why don't we want to scale subcell radii?

Reply:
oh goodness I don't remember, I think this came out of when I manually checked that all of the scaled results looked like what I would expect them to be. I'll double check, and if there is a good reason I'll make the comment more descriptive

- [ ] figure out why "do not scale radii for subcells"

Reply:
ok so I lied, I came across this because this matches the previous functionality so if i just did `radii *= scale_factor` for all radii, a previous test would fail. That's because "subcells" are represented as sphere groups, and so previously if someone had provided a scale factor, we wouldn't apply that to the agent of the sphere group's radii. I imagine it doesn't actually matter which way we do it, because I don't think the sphere group's agent radii matters? since the radii for each of the spheres of the sphere group is in the subpoints?